Story/Episode 5/co_052401_02.csv

```diff
@@ -1,3 +1,3 @@
 name01#0,"""Aika"""
-skit01#0,"""What!?<br>How is this possible, my servant!?"""
-skit01#1,"""What!?<br>How is this possible, my servant!?"""
+skit01#0,"""What?!<br>My familiar, vanquished... But how?!"""
```

Comment:
Make sure they're both the same.

Reply:
lmao I was very confused for a moment. Cursed interrobangs...